firebase_push_notifications/firebase_push_notifications/v1_0/handlers/handler.py

```diff
@@ -0,0 +1,30 @@
+"""Handler for push notifications."""
+import logging
+
+from aries_cloudagent.messaging.base_handler import (
+    BaseHandler,
+    BaseResponder,
+    RequestContext,
+)
+
+from ..messages.set_device_info import SetDeviceInfo
+from ..manager import save_device_token
+
+LOGGER = logging.getLogger(__name__)
+
+
+class SetDeviceInfoHandler(BaseHandler):
+    """Message handler class for push notifications."""
+
+    async def handle(self, context: RequestContext, responder: BaseResponder):
+        """Handle set-device-info message."""
+        connection_id = context.connection_record.connection_id
```

Comment:
Not sure if you need to run the linter/formatter

Reply:
The linter is run on all PR's and it's passing... So it should be fine.

Reply:
Yeah, resolved this after I wrote it. Not an issue.